CONTRIBUTING.md

```diff
@@ -1,7 +1,50 @@
 OpenProject is an open source project and we encourage you to help us out. We'd be happy if you do one of these things:
```

Comment:
Funnily enough, I had a different version which got lost in my git-fu.

I'd reword it to something along the lines of:

"If you want to help us, please read the following guidelines to contributing:"

Optionally we could reword the individual headlines to be more "pro-active", like "Creating tickets in OP", "Creating new features", "Creating hotfixes", etc.